src/main/java/org/opentripplanner/api/common/RoutingResource.java

```diff
@@ -890,10 +890,11 @@ protected RouteRequest buildRequest(MultivaluedMap<String, String> queryParamete
       request.journey().setModes(modes.getRequestModes());
     }
 
-    if (request.vehicleRental && bikeSpeed == null) {
-      //slower bike speed for bike sharing, based on empirical evidence from DC.
-      preferences.withBike(bike -> bike.setSpeed(4.3));
-    }
+    // TODO VIA (Hannes) - Add separate speeds for rental vehicles in preferences
```

Comment:
I guess this is intentional, too, isn't it?

Reply:
Same as above.